crates/hyperswitch_connectors/src/types.rs

```diff
@@ -1,11 +1,33 @@
 use hyperswitch_domain_models::{
-    router_data::RouterData, router_request_types::RefundsData,
-    router_response_types::RefundsResponseData,
+    router_data::RouterData,
+    router_flow_types::{Capture, PSync, Void},
+    router_request_types::{
+        PaymentsCancelData, PaymentsCaptureData, PaymentsSyncData, RefundsData,
+    },
+    router_response_types::{PaymentsResponseData, RefundsResponseData},
 };
 
+pub trait ForeignFrom<F> {
+    fn foreign_from(from: F) -> Self;
+}
+
+pub trait ForeignTryFrom<F>: Sized {
+    type Error;
+
+    fn foreign_try_from(from: F) -> Result<Self, Self::Error>;
+}
```

Comment:
Where are we using these two traits in the `hyperswitch_connectors` crate? I couldn't seem to find any usages?